content/operate/rs/7.4/clusters/logging/redis-slow-log.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: View Redis slow log
+alwaysopen: false
+categories:
+- docs
+- operate
+- rs
+description: null
+linkTitle: Slow log
+weight: $weight
```

Comment:
I noticed that several pages have `$weight`. Is this intended?

Reply:
It's a leftover setting from the old docs site, but it should be okay.